loader/CMakeLists.txt

```diff
@@ -141,6 +106,12 @@ if(WIN32)
     # Configure the file to include the versioning info
     # Place it in the current directory for check-in - so the GN build has up to date info
     configure_file(loader.rc.in ${CMAKE_CURRENT_LIST_DIR}/loader.rc)
+else()
+    # Used to make alloca() and secure_getenv() available
+    target_compile_definitions(loader_specific_options INTERFACE _GNU_SOURCE)
```

Comment:
Then is this needed if we have this case? "NOT (HAVE_SECURE_GETENV OR HAVE___SECURE_GETENV)"

Reply:
Yes, because `alloca` is a function made available with `_GNU_SOURCE`.